### ISSUE TITLE
Scheduling a group has unexpected results

### ISSUE THREAD

**Issue by WebSpider** [creativeprojects/resticprofile#381]
I realise the v2 config format being not-standard yet may have something to do with this, but I cannot seem to schedule a group backup/prune. Maybe I'm doing something wrong, maybe not. I'm looking to use this feature so the two jobs run in sequence: First make a backup of the DB, then make a backup of the files for the app, right after.

```yaml
version: 2

groups:
  mygroup:
    description: Database and file backup of App
    profiles:
      - app-database
      - app-files
    schedules:
      backup:
        at: "05:00"
        schedule-permission: user
      prune:
        at: monthly
        schedule-permission: user
profiles:
  default:
    <all my defaults go here>
  app-database:
    inherit: default
    backup:
      run-before: my-db-dump-script.sh
      run-after: my-db-dump-cleanup-script.sh
      source: my-db-dump-dir
    prune:
      max-unused: "3%"
  app-files:
    inherit: default
    exclude-file: files.exclude
    source:
      - /path/1
      - /path/2
   ```

When I then inpect what's going on, when I use `resticprofile profiles` it shows the 3 profiles and the group, but I cannot `show`, or `status` the group, because it is not a profile, and I cannot `schedule` the group, because the group has a schedule connected to it, and not the profiles.

When I move the schedule to the profiles, it works. But then I need to time how long it takes for my DB dump to complete, instead of chaining the profiles together.

**Comment by creativeprojects** [comment]
This is right, scheduling on v2 is not finished yet.
That's the last part to do, we're almost there 😉 

In the meantime, if you use resticprofile locks with a long waiting time, you should be able to do both backup without breaking the restic repository

**Comment by WebSpider** [comment]
You mean use the same lockfile for both backups?
That might be a nice solution indeed. I'll try that!

**Comment by creativeprojects** [comment]
Scheduling of groups is coming soon 😉 https://github.com/creativeprojects/resticprofile/pull/418

**Comment by creativeprojects** [comment]
Finally released in version `0.29.0`
